scripts/setup-jest.ts

```diff
@@ -6,6 +6,50 @@ class EventEmitter {
   public fire = (e: any) => this.listeners.forEach(listener => listener(e));
 }
 
+class Uri {
+  public static parse = jest.fn();
+  public static file = jest.fn();
+  public static joinPath = jest.fn();
+}
+
+const mockLanguageStatusItem = {
+  id: jest.fn(),
+  name: jest.fn(),
+  selector: jest.fn(),
+  severity: jest.fn(),
+  text: jest.fn(),
+  detail: jest.fn(),
+  busy: jest.fn(),
+  command: jest.fn(),
+  accessibilityInformation: jest.fn()
+};
+
+const mockCreateLanguageStatusItem = jest.fn();
+mockCreateLanguageStatusItem.mockReturnValue(mockLanguageStatusItem);
+
+enum LanguageStatusSeverity {
+  Information = 0,
+  Warning = 1,
+  Error = 2
+}
+
+enum TreeItemCollapsibleState {
+  None = 0,
+  Collapsed = 1,
+  Expanded = 2
+}
+
+const mockExtension = {
+  id: '1123',
+  extensionUri: Uri,
+  extensionPath: '123',
+  isActive: true,
+  packageJson: jest.fn(),
+  extensionKind: jest.fn(),
+  exports: jest.fn(),
+  activate: jest.fn()
+};
```

Comment:
hum...I'm a little worried about having these be outside teh getMockVSCode function.  Do this jest mocks get reset for each load of the mock vscode module?  I'll verify in QE

Reply:
turns out they do get reset so good to go 👍